lms/static/scripts/frontend_apps/config.js

```diff
@@ -155,6 +155,14 @@ import { createContext } from 'preact';
  * @prop {string} itemId
  */
 
+/**
+ * Debug information sent by the backend.
+ *
+ * @typedef DebugInfo
+ * @prop {string[]} [tags]
+ * @prop {Record<string,any>} [values]
```

Comment:
Should these properties be optional, or just the presence of the `debug` field in the config?

Reply:
I reckon combining different types of app it should be optional in all cases.